addon/components/flexberry-objectlistview.js

```diff
@@ -163,6 +163,33 @@ export default FlexberryBaseComponent.extend({
   */
   defaultLeftPadding: 10,
 
+  /**
+    Number page for search.
+
+    @property searchPageValue
+    @type Number
+  */
+  searchPageValue: undefined,
+
+  /**
+    Flag used for disable searchPageButton.
+
+    @property searchPageButtonReadonly
+    @type Boolean
+  */
+  searchPageButtonReadonly: computed('searchPageValue', function() {
```

Comment:
This property has the opposite meaning. If `true` button is active, otherwise disabled.